content/en/docs/v3.3/access-control-and-account-management/_index.md

```diff
@@ -8,6 +8,7 @@ weight: 12000
 
 icon: "/images/docs/v3.x/docs.svg"
 
+version: "v3.3"
```

Comment:
Why do we need to define version here as well, it's already defined in parent_index.md, right?